src/sentry/auth/authenticators/u2f.py

```diff
@@ -35,10 +48,19 @@ class U2fInterface(AuthenticatorInterface):
         "Chrome)."
     )
     allow_multi_enrollment = True
+    rp_id = urlparse(options.get("system.url-prefix")).hostname
+    rp = PublicKeyCredentialRpEntity(rp_id, "Sentry")
+    webauthn_registration_server = Fido2Server(rp)
+
+    def __init__(self, authenticator=None, status=EnrollmentStatus.EXISTING):
+        super().__init__(authenticator, status)
+
+        self.webauthn_authentication_server = U2FFido2Server(
+            app_id=self.u2f_app_id, rp={"id": self.rp_id, "name": "Sentry"}
+        )
+
     # rp is a relying party for webauthn, this would be sentry.io for SAAS
```

Comment:
should this comment be moved to where rp is declared?

Reply:
yes, i think that got moved once i merged in master